robomaker-sample-app-ci/src/aws-robomaker-sample-application-ci.ts

```diff
@@ -157,10 +157,15 @@ async function prepare_sources() {
       "README*",
       "roboMakerSettings.json"
     ];
-
+    
+    const execOptions: ExecOptions = {
+      cwd: path.join(WORKSPACE_DIRECTORY, ".."),
```

Comment:
If [this](https://github.com/aws-robotics/aws-robomaker-github-actions/pull/9/files#r488882730) is followed, are these changes still needed?

I also don't think we can always assume that the workspace is `..` relative to the GitHub action's working directory. I'm actually concerned that this change will break our current sample apps' CI.

Reply:
There are a lot of assumptions in the action itself. Indeed, it did not support multiple folders in working-dir since it use working-dir for the bundle filename. Here, I make an assumption that we want to store the bundle file one level before the working-dir. If this assumption is not valid then we need to add other params to the input, something like: bundle-folder and source-folder... There is an assumption that the default root destination is at the github-action checkout folder which does not work well when checking out multiple repos.

Reply:
Update: refactor getExecOptions and add getWorkingDirExecOptions and getWorkingDirParentExecOptions to make sure we are clear with which folder we want to run the command.